apps/web/src/features/photo-upload/index.ts

```
@@ -0,0 +1,9 @@
+export {
+  type ExistingPhotoPreview,
+  type NewPhotoPreview,
+  type PhotoPreview,
+  isNewPhotoPreview,
+  usePhotoUpload,
+} from './model/use-photo-upload';
+export { usePhotoUploadFeedback } from './model/use-photo-upload-feedback';
+export { default as PhotoUploader } from './ui/PhotoUploader';
```

ESLint

[error] Run autofix to sort these exports!

(simple-import-sort/exports)